day1/00_Organisation/01_Intro.md

```diff
@@ -34,3 +34,11 @@ Click on the link in your email to join the session
 * Operating system is CentOS
 * We use tmux to join your session and help you
 * Change the appearance of the terminal as you like
+
+!SLIDE smbullets
+# GitLab Instance
+
+* We are using a shared GitLab instance hosted on https://nws.netways.de/
+* The instance is a free-forever version
+* You should have received administrative rights for the instance
```

Comment:
participants only have regular user access.

Reply:
They can't look at the runner settings later into the training then, because that requires access to the admin menu...

Reply:
Thats for the Project, not the Instance. If you mean this: https://github.com/NETWAYS/gitlab-training/blob/main/day2/03_CI/03_Gitlab_CI.md?plain=1#L106

Reply:
Ah, yes, I did.
Let me change that really quick